src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not an SCSS file. Please change './styles/index.scss' to './styles/index.css'.